pkg/router/routes/api/config.go

```diff
@@ -6,14 +6,20 @@ import (
 	"strconv"
 
 	"github.com/plaenkler/ddns/pkg/config"
+	"github.com/plaenkler/ddns/pkg/util/limit"
```

Comment:
Why do you have a package path element called `util`? 
If I see it correctly, there are several packages in the folder. Shouldn't it be `utils` then, the plural?
I also think that this folder is superfluous and unnecessarily obscures the packages.

Reply:
Solved in [98dda37](https://github.com/Plaenkler/DDNS/pull/70/commits/98dda3783dfa196bfc4d3ed7d297b81c3336795e)